model/incoming_webhook.go

```diff
@@ -45,6 +46,23 @@ func (o *IncomingWebhook) ToJson() string {
 	return string(b)
 }
 
+func (o IncomingWebhook) String() string {
```

Comment:
I'll wait for that PR to be merged to use `prettyPrintStruct`.

Reply:
Just waiting for it to be rebased by the submitter, and then we should be good to merge the `prettyPrintStruct` PR. Once that is done, you can update this one and it can follow.

Reply:
Merged! :)

Reply:
Will update this later today